dallinger/command_line.py

```diff
@@ -806,21 +812,22 @@ def export(app, local):
             " --app " + app_name(id),
             shell=True)
 
-        dump_path = dump_database(id)
+    try:
+        subprocess.call([
+            "dropdb",
+            app_name(id),
+        ])
+    except Exception:
+        pass
 
-        try:
-            subprocess.check_call([
-                "pg_restore",
-                "--verbose",
-                "--no-owner",
-                "--clean",
-                "-d",
-                "dallinger",
-                os.path.join("data", id, "data.dump")
-            ])
-
-        except Exception:
-            pass
+    subprocess.call([
```

Comment:
Ideally, yes, but Heroku Postgres uses version 9.6 and I have 9.5 locally, which produces some harmless errors regarding `idle_in_transaction_session_timeout` and `row_security`, causing a non-zero exit status:

```
❯ dallinger export --app de304a7d-3ef4-4462-be0a-0cfefc1ecfb8

        ____        ____
       / __ \____ _/ / (_)___  ____ ____  _____
      / / / / __ `/ / / / __ \/ __ `/ _ \/ ___/
     / /_/ / /_/ / / / / / / / /_/ /  __/ /
    /_____/\__,_/_/_/_/_/ /_/\__, /\___/_/
                            /____/

                    Laboratory automation for
           the behavioral and social sciences.


❯❯ Preparing to export the data...
heroku-cli: Pulling postgresql-octagonal-37655 ---> dlgr-de304a7d
pg_dump: reading extensions
pg_dump: identifying extension members
pg_dump: reading schemas
pg_dump: reading user-defined tables
pg_dump: reading user-defined functions
pg_dump: reading user-defined types
pg_dump: reading procedural languages
pg_dump: reading user-defined aggregate functions
pg_dump: reading user-defined operators
pg_dump: reading user-defined access methods
pg_dump: reading user-defined operator classes
pg_dump: reading user-defined operator families
pg_dump: reading user-defined text search parsers
pg_dump: reading user-defined text search templates
pg_dump: reading user-defined text search dictionaries
pg_dump: reading user-defined text search configurations
pg_dump: reading user-defined foreign-data wrappers
pg_dump: reading user-defined foreign servers
pg_dump: reading default privileges
pg_dump: reading user-defined collations
pg_dump: reading user-defined conversions
pg_dump: reading type casts
pg_dump: reading transforms
pg_dump: reading table inheritance information
pg_dump: reading event triggers
pg_dump: finding extension tables
pg_dump: finding inheritance relationships
pg_dump: reading column info for interesting tables
pg_dump: finding the columns and types of table "public.pg_stat_statements"
pg_dump: finding the columns and types of table "public.participant"
pg_dump: finding default expressions of table "public.participant"
pg_dump: finding the columns and types of table "public.notification"
pg_dump: finding default expressions of table "public.notification"
pg_dump: finding the columns and types of table "public.network"
pg_dump: finding default expressions of table "public.network"
pg_dump: finding the columns and types of table "public.node"
pg_dump: finding default expressions of table "public.node"
pg_dump: finding the columns and types of table "public.question"
pg_dump: finding default expressions of table "public.question"
pg_dump: finding the columns and types of table "public.vector"
pg_dump: finding default expressions of table "public.vector"
pg_dump: finding the columns and types of table "public.info"
pg_dump: finding default expressions of table "public.info"
pg_dump: finding the columns and types of table "public.transmission"
pg_dump: finding default expressions of table "public.transmission"
pg_dump: finding the columns and types of table "public.transformation"
pg_dump: finding default expressions of table "public.transformation"
pg_dump: finding the columns and types of table "public.psiturk"
pg_dump: flagging inherited columns in subtables
pg_dump: reading indexes
pg_dump: reading indexes for table "public.participant"
pg_dump: reading indexes for table "public.notification"
pg_dump: reading indexes for table "public.network"
pg_dump: reading indexes for table "public.node"
pg_dump: reading indexes for table "public.question"
pg_dump: reading indexes for table "public.vector"
pg_dump: reading indexes for table "public.info"
pg_dump: reading indexes for table "public.transmission"
pg_dump: reading indexes for table "public.transformation"
pg_dump: reading indexes for table "public.psiturk"
pg_dump: reading constraints
pg_dump: reading foreign key constraints for table "public.participant"
pg_dump: reading foreign key constraints for table "public.network"
pg_dump: reading foreign key constraints for table "public.node"
pg_dump: reading foreign key constraints for table "public.question"
pg_dump: reading foreign key constraints for table "public.vector"
pg_dump: reading foreign key constraints for table "public.info"
pg_dump: reading foreign key constraints for table "public.transmission"
pg_dump: reading foreign key constraints for table "public.transformation"
pg_dump: reading triggers
pg_dump: reading triggers for table "public.participant"
pg_dump: reading triggers for table "public.network"
pg_dump: reading triggers for table "public.node"
pg_dump: reading triggers for table "public.question"
pg_dump: reading triggers for table "public.vector"
pg_dump: reading triggers for table "public.info"
pg_dump: reading triggers for table "public.transmission"
pg_dump: reading triggers for table "public.transformation"
pg_dump: reading rewrite rules
pg_dump: reading policies
pg_dump: reading row security enabled for table "public.pg_stat_statements"
pg_dump: reading policies for table "public.pg_stat_statements"
pg_dump: reading row security enabled for table "public.participant_id_seq"
pg_dump: reading policies for table "public.participant_id_seq"
pg_dump: reading row security enabled for table "public.participant"
pg_dump: reading policies for table "public.participant"
pg_dump: reading row security enabled for table "public.notification_id_seq"
pg_dump: reading policies for table "public.notification_id_seq"
pg_dump: reading row security enabled for table "public.notification"
pg_dump: reading policies for table "public.notification"
pg_dump: reading row security enabled for table "public.network_id_seq"
pg_dump: reading policies for table "public.network_id_seq"
pg_dump: reading row security enabled for table "public.network"
pg_dump: reading policies for table "public.network"
pg_dump: reading row security enabled for table "public.node_id_seq"
pg_dump: reading policies for table "public.node_id_seq"
pg_dump: reading row security enabled for table "public.node"
pg_dump: reading policies for table "public.node"
pg_dump: reading row security enabled for table "public.question_id_seq"
pg_dump: reading policies for table "public.question_id_seq"
pg_dump: reading row security enabled for table "public.question"
pg_dump: reading policies for table "public.question"
pg_dump: reading row security enabled for table "public.vector_id_seq"
pg_dump: reading policies for table "public.vector_id_seq"
pg_dump: reading row security enabled for table "public.vector"
pg_dump: reading policies for table "public.vector"
pg_dump: reading row security enabled for table "public.info_id_seq"
pg_dump: reading policies for table "public.info_id_seq"
pg_dump: reading row security enabled for table "public.info"
pg_dump: reading policies for table "public.info"
pg_dump: reading row security enabled for table "public.transmission_id_seq"
pg_dump: reading policies for table "public.transmission_id_seq"
pg_dump: reading row security enabled for table "public.transmission"
pg_dump: reading policies for table "public.transmission"
pg_dump: reading row security enabled for table "public.transformation_id_seq"
pg_dump: reading policies for table "public.transformation_id_seq"
pg_dump: reading row security enabled for table "public.transformation"
pg_dump: reading policies for table "public.transformation"
pg_dump: reading row security enabled for table "public.psiturk"
pg_dump: reading policies for table "public.psiturk"
pg_dump: reading large objects
pg_dump: reading dependency data
pg_dump: saving encoding = UTF8
pg_dump: saving standard_conforming_strings = on
pg_dump: saving database definition
pg_dump: dumping contents of table "public.info"
pg_dump: dumping contents of table "public.network"
pg_dump: dumping contents of table "public.node"
pg_dump: dumping contents of table "public.notification"
pg_dump: dumping contents of table "public.participant"
pg_dump: dumping contents of table "public.psiturk"
pg_dump: dumping contents of table "public.question"
pg_dump: dumping contents of table "public.transformation"
pg_dump: dumping contents of table "public.transmission"
pg_dump: dumping contents of table "public.vector"
pg_restore: connecting to database for restore
pg_restore: [archiver (db)] Error while INITIALIZING:
pg_restore: [archiver (db)] could not execute query: ERROR:  unrecognized configuration parameter "idle_in_transaction_session_timeout"
    Command was: SET idle_in_transaction_session_timeout = 0;

pg_restore: [archiver (db)] could not execute query: ERROR:  unrecognized configuration parameter "row_security"
    Command was: SET row_security = off;

pg_restore: creating SCHEMA "public"
pg_restore: creating COMMENT "SCHEMA public"
pg_restore: creating EXTENSION "plpgsql"
pg_restore: creating COMMENT "EXTENSION plpgsql"
pg_restore: creating EXTENSION "pg_stat_statements"
pg_restore: creating COMMENT "EXTENSION pg_stat_statements"
pg_restore: creating TYPE "public.participant_status"
pg_restore: creating TYPE "public.transmission_status"
pg_restore: creating TABLE "public.info"
pg_restore: creating SEQUENCE "public.info_id_seq"
pg_restore: creating SEQUENCE OWNED BY "public.info_id_seq"
pg_restore: creating TABLE "public.network"
pg_restore: creating SEQUENCE "public.network_id_seq"
pg_restore: creating SEQUENCE OWNED BY "public.network_id_seq"
pg_restore: creating TABLE "public.node"
pg_restore: creating SEQUENCE "public.node_id_seq"
pg_restore: creating SEQUENCE OWNED BY "public.node_id_seq"
pg_restore: creating TABLE "public.notification"
pg_restore: creating SEQUENCE "public.notification_id_seq"
pg_restore: creating SEQUENCE OWNED BY "public.notification_id_seq"
pg_restore: creating TABLE "public.participant"
pg_restore: creating SEQUENCE "public.participant_id_seq"
pg_restore: creating SEQUENCE OWNED BY "public.participant_id_seq"
pg_restore: creating TABLE "public.psiturk"
pg_restore: creating TABLE "public.question"
pg_restore: creating SEQUENCE "public.question_id_seq"
pg_restore: creating SEQUENCE OWNED BY "public.question_id_seq"
pg_restore: creating TABLE "public.transformation"
pg_restore: creating SEQUENCE "public.transformation_id_seq"
pg_restore: creating SEQUENCE OWNED BY "public.transformation_id_seq"
pg_restore: creating TABLE "public.transmission"
pg_restore: creating SEQUENCE "public.transmission_id_seq"
pg_restore: creating SEQUENCE OWNED BY "public.transmission_id_seq"
pg_restore: creating TABLE "public.vector"
pg_restore: creating SEQUENCE "public.vector_id_seq"
pg_restore: creating SEQUENCE OWNED BY "public.vector_id_seq"
pg_restore: creating DEFAULT "public.info id"
pg_restore: creating DEFAULT "public.network id"
pg_restore: creating DEFAULT "public.node id"
pg_restore: creating DEFAULT "public.notification id"
pg_restore: creating DEFAULT "public.participant id"
pg_restore: creating DEFAULT "public.question id"
pg_restore: creating DEFAULT "public.transformation id"
pg_restore: creating DEFAULT "public.transmission id"
pg_restore: creating DEFAULT "public.vector id"
pg_restore: processing data for table "public.info"
pg_restore: executing SEQUENCE SET info_id_seq
pg_restore: processing data for table "public.network"
pg_restore: executing SEQUENCE SET network_id_seq
pg_restore: processing data for table "public.node"
pg_restore: executing SEQUENCE SET node_id_seq
pg_restore: processing data for table "public.notification"
pg_restore: executing SEQUENCE SET notification_id_seq
pg_restore: processing data for table "public.participant"
pg_restore: executing SEQUENCE SET participant_id_seq
pg_restore: processing data for table "public.psiturk"
pg_restore: processing data for table "public.question"
pg_restore: executing SEQUENCE SET question_id_seq
pg_restore: processing data for table "public.transformation"
pg_restore: executing SEQUENCE SET transformation_id_seq
pg_restore: processing data for table "public.transmission"
pg_restore: executing SEQUENCE SET transmission_id_seq
pg_restore: processing data for table "public.vector"
pg_restore: executing SEQUENCE SET vector_id_seq
pg_restore: creating CONSTRAINT "public.info info_pkey"
pg_restore: creating CONSTRAINT "public.network network_pkey"
pg_restore: creating CONSTRAINT "public.node node_pkey"
pg_restore: creating CONSTRAINT "public.notification notification_pkey"
pg_restore: creating CONSTRAINT "public.participant participant_pkey"
pg_restore: creating CONSTRAINT "public.psiturk psiturk_pkey"
pg_restore: creating CONSTRAINT "public.question question_pkey"
pg_restore: creating CONSTRAINT "public.transformation transformation_pkey"
pg_restore: creating CONSTRAINT "public.transmission transmission_pkey"
pg_restore: creating CONSTRAINT "public.vector vector_pkey"
pg_restore: creating INDEX "public.ix_info_failed"
pg_restore: creating INDEX "public.ix_info_id"
pg_restore: creating INDEX "public.ix_info_network_id"
pg_restore: creating INDEX "public.ix_info_origin_id"
pg_restore: creating INDEX "public.ix_network_failed"
pg_restore: creating INDEX "public.ix_network_full"
pg_restore: creating INDEX "public.ix_network_id"
pg_restore: creating INDEX "public.ix_network_role"
pg_restore: creating INDEX "public.ix_node_failed"
pg_restore: creating INDEX "public.ix_node_id"
pg_restore: creating INDEX "public.ix_node_network_id"
pg_restore: creating INDEX "public.ix_node_participant_id"
pg_restore: creating INDEX "public.ix_notification_failed"
pg_restore: creating INDEX "public.ix_notification_id"
pg_restore: creating INDEX "public.ix_participant_assignment_id"
pg_restore: creating INDEX "public.ix_participant_failed"
pg_restore: creating INDEX "public.ix_participant_id"
pg_restore: creating INDEX "public.ix_participant_status"
pg_restore: creating INDEX "public.ix_participant_unique_id"
pg_restore: creating INDEX "public.ix_question_failed"
pg_restore: creating INDEX "public.ix_question_id"
pg_restore: creating INDEX "public.ix_transformation_failed"
pg_restore: creating INDEX "public.ix_transformation_id"
pg_restore: creating INDEX "public.ix_transformation_info_in_id"
pg_restore: creating INDEX "public.ix_transformation_info_out_id"
pg_restore: creating INDEX "public.ix_transformation_network_id"
pg_restore: creating INDEX "public.ix_transformation_node_id"
pg_restore: creating INDEX "public.ix_transmission_destination_id"
pg_restore: creating INDEX "public.ix_transmission_failed"
pg_restore: creating INDEX "public.ix_transmission_id"
pg_restore: creating INDEX "public.ix_transmission_info_id"
pg_restore: creating INDEX "public.ix_transmission_network_id"
pg_restore: creating INDEX "public.ix_transmission_origin_id"
pg_restore: creating INDEX "public.ix_transmission_status"
pg_restore: creating INDEX "public.ix_transmission_vector_id"
pg_restore: creating INDEX "public.ix_vector_destination_id"
pg_restore: creating INDEX "public.ix_vector_failed"
pg_restore: creating INDEX "public.ix_vector_id"
pg_restore: creating INDEX "public.ix_vector_network_id"
pg_restore: creating INDEX "public.ix_vector_origin_id"
pg_restore: creating FK CONSTRAINT "public.info info_network_id_fkey"
pg_restore: creating FK CONSTRAINT "public.info info_origin_id_fkey"
pg_restore: creating FK CONSTRAINT "public.node node_network_id_fkey"
pg_restore: creating FK CONSTRAINT "public.node node_participant_id_fkey"
pg_restore: creating FK CONSTRAINT "public.question question_participant_id_fkey"
pg_restore: creating FK CONSTRAINT "public.transformation transformation_info_in_id_fkey"
pg_restore: creating FK CONSTRAINT "public.transformation transformation_info_out_id_fkey"
pg_restore: creating FK CONSTRAINT "public.transformation transformation_network_id_fkey"
pg_restore: creating FK CONSTRAINT "public.transformation transformation_node_id_fkey"
pg_restore: creating FK CONSTRAINT "public.transmission transmission_destination_id_fkey"
pg_restore: creating FK CONSTRAINT "public.transmission transmission_info_id_fkey"
pg_restore: creating FK CONSTRAINT "public.transmission transmission_network_id_fkey"
pg_restore: creating FK CONSTRAINT "public.transmission transmission_origin_id_fkey"
pg_restore: creating FK CONSTRAINT "public.transmission transmission_vector_id_fkey"
pg_restore: creating FK CONSTRAINT "public.vector vector_destination_id_fkey"
pg_restore: creating FK CONSTRAINT "public.vector vector_network_id_fkey"
pg_restore: creating FK CONSTRAINT "public.vector vector_origin_id_fkey"
pg_restore: setting owner and privileges for DATABASE "da1hdkq87o2qfm"
pg_restore: setting owner and privileges for SCHEMA "public"
pg_restore: setting owner and privileges for COMMENT "SCHEMA public"
pg_restore: setting owner and privileges for EXTENSION "plpgsql"
pg_restore: setting owner and privileges for COMMENT "EXTENSION plpgsql"
pg_restore: setting owner and privileges for EXTENSION "pg_stat_statements"
pg_restore: setting owner and privileges for COMMENT "EXTENSION pg_stat_statements"
pg_restore: setting owner and privileges for TYPE "public.participant_status"
pg_restore: setting owner and privileges for TYPE "public.transmission_status"
pg_restore: setting owner and privileges for TABLE "public.info"
pg_restore: setting owner and privileges for SEQUENCE "public.info_id_seq"
pg_restore: setting owner and privileges for SEQUENCE OWNED BY "public.info_id_seq"
pg_restore: setting owner and privileges for TABLE "public.network"
pg_restore: setting owner and privileges for SEQUENCE "public.network_id_seq"
pg_restore: setting owner and privileges for SEQUENCE OWNED BY "public.network_id_seq"
pg_restore: setting owner and privileges for TABLE "public.node"
pg_restore: setting owner and privileges for SEQUENCE "public.node_id_seq"
pg_restore: setting owner and privileges for SEQUENCE OWNED BY "public.node_id_seq"
pg_restore: setting owner and privileges for TABLE "public.notification"
pg_restore: setting owner and privileges for SEQUENCE "public.notification_id_seq"
pg_restore: setting owner and privileges for SEQUENCE OWNED BY "public.notification_id_seq"
pg_restore: setting owner and privileges for TABLE "public.participant"
pg_restore: setting owner and privileges for SEQUENCE "public.participant_id_seq"
pg_restore: setting owner and privileges for SEQUENCE OWNED BY "public.participant_id_seq"
pg_restore: setting owner and privileges for TABLE "public.psiturk"
pg_restore: setting owner and privileges for TABLE "public.question"
pg_restore: setting owner and privileges for SEQUENCE "public.question_id_seq"
pg_restore: setting owner and privileges for SEQUENCE OWNED BY "public.question_id_seq"
pg_restore: setting owner and privileges for TABLE "public.transformation"
pg_restore: setting owner and privileges for SEQUENCE "public.transformation_id_seq"
pg_restore: setting owner and privileges for SEQUENCE OWNED BY "public.transformation_id_seq"
pg_restore: setting owner and privileges for TABLE "public.transmission"
pg_restore: setting owner and privileges for SEQUENCE "public.transmission_id_seq"
pg_restore: setting owner and privileges for SEQUENCE OWNED BY "public.transmission_id_seq"
pg_restore: setting owner and privileges for TABLE "public.vector"
pg_restore: setting owner and privileges for SEQUENCE "public.vector_id_seq"
pg_restore: setting owner and privileges for SEQUENCE OWNED BY "public.vector_id_seq"
pg_restore: setting owner and privileges for DEFAULT "public.info id"
pg_restore: setting owner and privileges for DEFAULT "public.network id"
pg_restore: setting owner and privileges for DEFAULT "public.node id"
pg_restore: setting owner and privileges for DEFAULT "public.notification id"
pg_restore: setting owner and privileges for DEFAULT "public.participant id"
pg_restore: setting owner and privileges for DEFAULT "public.question id"
pg_restore: setting owner and privileges for DEFAULT "public.transformation id"
pg_restore: setting owner and privileges for DEFAULT "public.transmission id"
pg_restore: setting owner and privileges for DEFAULT "public.vector id"
pg_restore: setting owner and privileges for TABLE DATA "public.info"
pg_restore: setting owner and privileges for SEQUENCE SET "public.info_id_seq"
pg_restore: setting owner and privileges for TABLE DATA "public.network"
pg_restore: setting owner and privileges for SEQUENCE SET "public.network_id_seq"
pg_restore: setting owner and privileges for TABLE DATA "public.node"
pg_restore: setting owner and privileges for SEQUENCE SET "public.node_id_seq"
pg_restore: setting owner and privileges for TABLE DATA "public.notification"
pg_restore: setting owner and privileges for SEQUENCE SET "public.notification_id_seq"
pg_restore: setting owner and privileges for TABLE DATA "public.participant"
pg_restore: setting owner and privileges for SEQUENCE SET "public.participant_id_seq"
pg_restore: setting owner and privileges for TABLE DATA "public.psiturk"
pg_restore: setting owner and privileges for TABLE DATA "public.question"
pg_restore: setting owner and privileges for SEQUENCE SET "public.question_id_seq"
pg_restore: setting owner and privileges for TABLE DATA "public.transformation"
pg_restore: setting owner and privileges for SEQUENCE SET "public.transformation_id_seq"
pg_restore: setting owner and privileges for TABLE DATA "public.transmission"
pg_restore: setting owner and privileges for SEQUENCE SET "public.transmission_id_seq"
pg_restore: setting owner and privileges for TABLE DATA "public.vector"
pg_restore: setting owner and privileges for SEQUENCE SET "public.vector_id_seq"
pg_restore: setting owner and privileges for CONSTRAINT "public.info info_pkey"
pg_restore: setting owner and privileges for CONSTRAINT "public.network network_pkey"
pg_restore: setting owner and privileges for CONSTRAINT "public.node node_pkey"
pg_restore: setting owner and privileges for CONSTRAINT "public.notification notification_pkey"
pg_restore: setting owner and privileges for CONSTRAINT "public.participant participant_pkey"
pg_restore: setting owner and privileges for CONSTRAINT "public.psiturk psiturk_pkey"
pg_restore: setting owner and privileges for CONSTRAINT "public.question question_pkey"
pg_restore: setting owner and privileges for CONSTRAINT "public.transformation transformation_pkey"
pg_restore: setting owner and privileges for CONSTRAINT "public.transmission transmission_pkey"
pg_restore: setting owner and privileges for CONSTRAINT "public.vector vector_pkey"
pg_restore: setting owner and privileges for INDEX "public.ix_info_failed"
pg_restore: setting owner and privileges for INDEX "public.ix_info_id"
pg_restore: setting owner and privileges for INDEX "public.ix_info_network_id"
pg_restore: setting owner and privileges for INDEX "public.ix_info_origin_id"
pg_restore: setting owner and privileges for INDEX "public.ix_network_failed"
pg_restore: setting owner and privileges for INDEX "public.ix_network_full"
pg_restore: setting owner and privileges for INDEX "public.ix_network_id"
pg_restore: setting owner and privileges for INDEX "public.ix_network_role"
pg_restore: setting owner and privileges for INDEX "public.ix_node_failed"
pg_restore: setting owner and privileges for INDEX "public.ix_node_id"
pg_restore: setting owner and privileges for INDEX "public.ix_node_network_id"
pg_restore: setting owner and privileges for INDEX "public.ix_node_participant_id"
pg_restore: setting owner and privileges for INDEX "public.ix_notification_failed"
pg_restore: setting owner and privileges for INDEX "public.ix_notification_id"
pg_restore: setting owner and privileges for INDEX "public.ix_participant_assignment_id"
pg_restore: setting owner and privileges for INDEX "public.ix_participant_failed"
pg_restore: setting owner and privileges for INDEX "public.ix_participant_id"
pg_restore: setting owner and privileges for INDEX "public.ix_participant_status"
pg_restore: setting owner and privileges for INDEX "public.ix_participant_unique_id"
pg_restore: setting owner and privileges for INDEX "public.ix_question_failed"
pg_restore: setting owner and privileges for INDEX "public.ix_question_id"
pg_restore: setting owner and privileges for INDEX "public.ix_transformation_failed"
pg_restore: setting owner and privileges for INDEX "public.ix_transformation_id"
pg_restore: setting owner and privileges for INDEX "public.ix_transformation_info_in_id"
pg_restore: setting owner and privileges for INDEX "public.ix_transformation_info_out_id"
pg_restore: setting owner and privileges for INDEX "public.ix_transformation_network_id"
pg_restore: setting owner and privileges for INDEX "public.ix_transformation_node_id"
pg_restore: setting owner and privileges for INDEX "public.ix_transmission_destination_id"
pg_restore: setting owner and privileges for INDEX "public.ix_transmission_failed"
pg_restore: setting owner and privileges for INDEX "public.ix_transmission_id"
pg_restore: setting owner and privileges for INDEX "public.ix_transmission_info_id"
pg_restore: setting owner and privileges for INDEX "public.ix_transmission_network_id"
pg_restore: setting owner and privileges for INDEX "public.ix_transmission_origin_id"
pg_restore: setting owner and privileges for INDEX "public.ix_transmission_status"
pg_restore: setting owner and privileges for INDEX "public.ix_transmission_vector_id"
pg_restore: setting owner and privileges for INDEX "public.ix_vector_destination_id"
pg_restore: setting owner and privileges for INDEX "public.ix_vector_failed"
pg_restore: setting owner and privileges for INDEX "public.ix_vector_id"
pg_restore: setting owner and privileges for INDEX "public.ix_vector_network_id"
pg_restore: setting owner and privileges for INDEX "public.ix_vector_origin_id"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.info info_network_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.info info_origin_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.node node_network_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.node node_participant_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.question question_participant_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.transformation transformation_info_in_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.transformation transformation_info_out_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.transformation transformation_network_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.transformation transformation_node_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.transmission transmission_destination_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.transmission transmission_info_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.transmission transmission_network_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.transmission transmission_origin_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.transmission transmission_vector_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.vector vector_destination_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.vector vector_network_id_fkey"
pg_restore: setting owner and privileges for FK CONSTRAINT "public.vector vector_origin_id_fkey"
WARNING: errors ignored on restore: 2
Traceback (most recent call last):
  File "/Users/jordansuchow/.virtualenvs/dallinger/bin/dallinger", line 11, in <module>
    load_entry_point('dallinger', 'console_scripts', 'dallinger')()
  File "/Users/jordansuchow/.virtualenvs/dallinger/lib/python2.7/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/Users/jordansuchow/.virtualenvs/dallinger/lib/python2.7/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/Users/jordansuchow/.virtualenvs/dallinger/lib/python2.7/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/jordansuchow/.virtualenvs/dallinger/lib/python2.7/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/jordansuchow/.virtualenvs/dallinger/lib/python2.7/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/Users/jordansuchow/Dropbox/Berkeley/Projects/Current/Dallinger/dallinger/command_line.py", line 770, in export
    export_data(id, local)
  File "/Users/jordansuchow/Dropbox/Berkeley/Projects/Current/Dallinger/dallinger/command_line.py", line 829, in export_data
    app_name(id),
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['heroku', 'pg:pull', 'DATABASE_URL', 'dlgr-de304a7d', '--app', 'dlgr-de304a7d']' returned non-zero exit status 1
```